services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/modules/long_running_tasks_utils.py

```diff
@@ -40,3 +46,22 @@ async def run_before_shutdown_actions(
                     container_name,
                     exc_info=True,
                 )
+
+
+async def ensure_read_permissions_on_user_service_data(
+    mounted_volumes: MountedVolumes,
+) -> None:
+    # Makes sure sidecar has access to all files in the user services.
+    # The user could have removed read permissions form a file, which will cause an error.
+
+    # NOTE: command runs inside self container since the user service container might not always be running
+    self_container = os.environ["HOSTNAME"]
+    for path_to_store in (  # apply changes to otuputs and all state folders
+        *mounted_volumes.disk_state_paths_iter(),
+        mounted_volumes.disk_outputs_path,
+    ):
+        await run_command_in_container(
+            self_container,
+            command=f"chmod -R go+rX {path_to_store}",
```

Comment:
Warning: `path_to_store` does not need to be in between quotes for safety ? That is why i like using command as a list instead of a string